news/3833.bugfix

```diff
@@ -0,0 +1 @@
+Make PloneSite have IContentish again. @Akshat2Jain @jaroel
```

Comment:
I think this is a feature. Not a bugfix.

Reply:
I don't have a preference, I just need people to be aware.
@mauritsvanrees any preference so we can move forward?

Reply:
Feels more like a regression fix to me, so let's stick with bugfix.